dev_tools/scripts/es_test.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
 
+num=100
+
+if [ ! -z $1 ]; then
+  num=$1
+fi
```

Comment:
```suggestion
if [ -n $1 ]; then
  num=$1
fi
```

Reply:
Or even better (as in more portable/safer) `[ -n "$1"  ]`.